pom.xml

```diff
@@ -25,7 +25,7 @@
     <dependency>
       <groupId>com.amazonaws</groupId>
       <artifactId>aws-java-sdk</artifactId>
-      <version>1.11.126</version>
+      <version>1.11.197</version>
```

Comment:
I would recommend to use https://plugins.jenkins.io/aws-java-sdk instead so that the plugin does not bundle it's own library copy

Reply:
didn't get what you mean here. Instead of bumping the required version you want me to do what exactly?

Reply:
To depend on an API plugin instead of a lib directly. No worries, it can be done in a follow-up PR